tests/data/Makefile.inc

```diff
@@ -55,7 +55,7 @@ test280 test281 test282 test283 test284 test285 test286 test287 test288 \
 test289 test290 test291 test292 test293 test294 test295 test296 test297 \
 test298 test299 test300 test301 test302 test303 test304 test305 test306 \
 test307 test308 test309 test310 test311 test312 test313 test314 test315 \
-                                test320 test321 test322 test323 test324 \
+test316 test317                 test320 test321 test322 test323 test324 \
```

Comment:
i rebased this and renamed the tests to avoid conflicts 314 => 316 and 315 => 317

Reply:
Cool, thanks!